src/cryptoadvance/specter/templates/wallet_addresses.html

```diff
@@ -48,45 +51,75 @@ <h1><small style="line-height:30px">Total balance:<br></small><span style="color
 	( {{ "%0.8f" % wallet.balance["trusted"] | float }} confirmed, {{"%0.8f" % wallet.balance["untrusted_pending"] | float}} pending )
 	{% endif %}
 </small></h1>
-<br>
-<h1>Address History</h1>
-{% for address in wallet.addresses | reverse %}
-{% set addrlabel = wallet.getaddressname(address, wallet.addresses | length - loop.index0 - 1) %}
+
+<div style="padding: 0 30px; width: 100%;">
+	<div class="row padded" style="width: intrinsic; margin: auto;">
+		<a href="./?all=True&view={{viewtype}}" class="btn radio left {% if alladdresses %}checked{% endif %}">History</a>
+		<a href="./?all=False&view={{viewtype}}" class="btn radio right {% if alladdresses != True %}checked{% endif %}">UTXO</a>
+	</div>
+	<span>View Type: </span><br>
+	<div class="dropdown" style="margin-top: 5px;">
```

Comment:
I'm not sure why it's worth the effort? The only difference seem to be that the adresses disappear?

Reply:
My thought with adding that was to allow grouping multiple addresses with the same label, creating an "account"-like view within the wallet. So if you want to manage multiple "accounts" within the same wallet you give each address the name of the account as its label and then you can see all its addresses and outputs grouped together.
Another way to implement an "accounts" feature is by utilizing the account system of the BIP32 derivation path, but this would be much more different and more complicated.

Reply:
Ah, got it, neat :-). Really like that.